about.html

```diff
@@ -0,0 +1,100 @@
+      <a class="nav-item" href="#"><i class="fa fa-facebook fa_custom"></i></a>
+      <a class="nav-item" href="#"><i class="fa fa-twitter"></i></a>
+      <a class="nav-item" href="#">English</a>
+      <a class="nav-item" href="index.html">Homepage</a>
+      <a class="nav-item" href="#">Logout</a>
+    </nav>
+    <nav class="navbar-two">
+      <div class="navbar-header">
+        <div class="humburger">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </div>
+        <a id="header-logo" href="index.html"><img src="./img/YCI-LOGO2.jpg" alt="YCI Logo"></a>
+      </div>
+      <div class="collapse navbar-collapse" id="myNavbar">
+        <a class="link" href="index.html">Home</a>
+        <a class="link" href="about.html">About</a>
+        <a class="link" href="#">Program</a>
+        <a class="link" href="#">Join</a>
+        <a class="link" href="#">Sponsors</a>
+        <a class="link" href="#">News</a>
+        <a href="#" id="campaign" class="link">YCI LAUNCH</a>
+      </div>
+    </nav>
+  </header>
+
+  <section class="about-container">
+    <p class="greetings" id="about-greetings">"Hello! Welcome to YCI"</p>
+    <h1 class="about-h1">YOUNG CODERS INITIATIVE<br>2022</h1>
+    <div class="program-description">
+      <p class="detail-description">
+        Young Coders Initiative (YCI) was established to help seconandary school students learn 
+        how to code and create Software programs from a tender age. Young Coders Initiative 
+        aims at equipping secondary school students with skills and knowledge in the field of computer science. 
+        YCI launches in December, 2022, in Malawi.
+      </p>
+      <p class="futher-info">
+        Please contact us via email for any further information about Young Coders Initiative and it's National Launch!
+        <br><span id="email-link">youngcodersyci@gmail.com</span>
+      </p>
+    </div>
+    <div class="past-events">
+      <div class="about-logo">
+        <h2 class="past-event-header">About Young Coders Initiative</h2>
+        <p class="past-event-para">
+          Young Coders Initiative aims at equipping secondary school students with skills and knowledge in the field of computer science. 
+          The skills and knowledge will enable them engage in businesses related to ICT.    
+        </p>
+        <div class="info-about-logo">
+          <img src="/img/YCI-LOGO2.jpg" alt="YCI Logo">
+        </div>
+      </div>
+      <h2 class="past-event-header">See the first Pioneers Young Coders Initiative (YCI)</h2>
+      <p class="past-event-para">
+        Take a look at the snippets of First students of Young Coders Initiative.
+      </p>
+      <div class="event-media">
+        <div class="container-20">
+          <div class="text-20">2021</div>
+        </div>
+        <div class="container-21">
+          <div class="text-21">2022</div>
+        </div>
+      </div>
```

Comment:
Kindly address the unwanted overflow that is happening on your mobile design 😎

![image](https://user-images.githubusercontent.com/15807343/177009652-9c72b3f9-1146-4332-8ffa-4b0d7253d316.png)

As you will see see, event-media is responsible of the issue 😥